### PR TITLE
Upgrade, improve security move to main catalog

### PR DESCRIPTION
**Pull request by paurosello**
Upgrade to latest release
Improve security by executing as user with privileges and not root
Switch to main catalog

### CODE REVIEW

File: helm/k8s-dns-node-cache-app/values.yaml [hunk 1]
```diff
@@ -9,7 +9,7 @@ serviceType: managed
 image:
   registry: quay.io
   name: giantswarm/k8s-dns-node-cache
-  tag: 1.21.4
+  tag: 1.22.13
```
Comment (whites11) [inline helm/k8s-dns-node-cache-app/values.yaml:12]: You might want to bump here as well https://github.com/giantswarm/k8s-dns-node-cache-app/blob/main/helm/k8s-dns-node-cache-app/Chart.yaml#L2

**Comment by whites11** [comment]
If we own it, we should add the team annotation like this https://github.com/giantswarm/aws-operator/blob/master/helm/aws-operator/Chart.yaml#L8

**Comment by whites11** [comment]
Also please ensure we are following the advice here https://github.com/giantswarm/roadmap/issues/1436

File: helm/k8s-dns-node-cache-app/values.yaml (hunk 1 above)
Comment (paurosello) [reply helm/k8s-dns-node-cache-app/values.yaml:12]: Done

**Comment by paurosello** [comment]
We have the new labels in place too

Screenshot 2022-10-28 at 16 23 01

**Comment by paurosello** [comment]
PR in https://github.com/giantswarm/k8s-dns-node-cache-app/pull/15

**Review by whites11 — APPROVED** [review]
Looking good, man!

**Pull request merged**